<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lab14Class</class>
 <widget class="QMainWindow" name="Lab14Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lab14</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="allRecordingsLabel">
        <property name="font">
         <font>
          <family>Eras Bold ITC</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>All recordings</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QListWidget" name="recordingListWidget"/>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QPushButton" name="addButton">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="updateButton">
            <property name="text">
             <string>Update</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="deleteButton">
            <property name="text">
             <string>Delete</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="undoButton">
            <property name="text">
             <string>Undo</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QPushButton" name="redoButton">
            <property name="text">
             <string>Redo</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QPushButton" name="modeAButton">
            <property name="text">
             <string>mode A</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QPushButton" name="modeBButton">
            <property name="text">
             <string>mode B</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="recordingDetailsLabel">
        <property name="font">
         <font>
          <family>Eras Bold ITC</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Recording details</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="titleLabel">
          <property name="text">
           <string>Title</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="titleLineEdit"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="locationLabel">
          <property name="text">
           <string>Location</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="locationLineEdit"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="timeOfCreationLabel">
          <property name="text">
           <string>Time of Creation</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="timeOfCreationLineEdit"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="timesAccessedLabel">
          <property name="text">
           <string>Times accessed</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="timesAccessedLineEdit"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="footagePreviewLabel">
          <property name="text">
           <string>Footage preview</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="footagePreviewLineEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Eras Bold ITC</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Watch list</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QListWidget" name="watchListWidget"/>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="4" column="0">
           <widget class="QPushButton" name="addToWatchListButton">
            <property name="text">
             <string>Add to watch list</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QPushButton" name="lookAtWatchListButton">
            <property name="text">
             <string>Look at watch list</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="removeFilterButton">
            <property name="text">
             <string>Remove Filter</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="filterButton">
            <property name="text">
             <string>Filter</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="nextWatchListButton">
            <property name="text">
             <string>Next</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QPushButton" name="watchListTableViewButton">
            <property name="text">
             <string>Open watch list in TableView</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Eras Bold ITC</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Filter details</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="1">
         <widget class="QLineEdit" name="watchListTimesAccessedLineEdit"/>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="watchListLocationLineEdit"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="watchListTimesAccessedLabel">
          <property name="text">
           <string>Times accessed</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="watchListlocationLabel">
          <property name="text">
           <string>Location</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Lab14.qrc"/>
 </resources>
 <connections/>
</ui>
